src/components/WorkoutDisplayCard/index.tsx

```diff
@@ -32,11 +32,7 @@ export default function WorkoutDisplayCard() {
     }
     async function getAllExercise() {
       console.log('all exercises');
```

Comment:
Do we still need this console.log?

Reply:
idk you were meant to tell me, "code reviewer"